packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,223 @@
 # @fluentui/react-persona Spec
 
+Convergence epic issue: #24213
+
 ## Background
 
-_Description and use cases of this component_
+A Persona is a visual representation of a person or status that showcases an Avatar, PresenceBadge, or an Avatar with a PresenceBadge. This composite component is not a card, therefore it will not provide a visual board or a popup when the user hovers the Avatar.
+
+Persona is used in PeoplePicker, Team's left rail and menus, chiclets, and Card.
 
 ## Prior Art
 
-_Include background research done for this component_
+- OpenUI research: [Avatar/Persona](https://open-ui.org/components/avatar.research)
+- v7/v8: [link](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+  - > Note: v8's Persona is a combination of v9's Persona and Avatar (v8's Persona has a sub-component PersonaCoin that would count as Avatar, but is not exposed.). In v9 we've taken the approach of having a single component Avatar and another component Persona.
+- v0: Does not have equivalent component.
+
+## Anatomy
+
+![Anatomy](./etc/images/anatomy.png)
+
+### Avatar, PresenceBadge, and Avatar + PresenceBadge
+
+v8's Persona has a sub-component `PersonaCoin`, the equivalent in v9 is an Avatar. Currently Avatar is able to display a PresenceBadge, but this behavior will be deprecated in favor of Persona having this behavior.
+
+Persona has a slot for an `Avatar` and `PresenceBadge`, giving the user the ability to display an Avatar, a PresenceBadge, or Combine them into an Avatar with presence.
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
```

Comment:
as v8, sorry I'll update that.

Reply:
Hmm I think comparisons to v8 should be limited to the Prior Art section. Otherwise, this spec should just describe how Persona works, without using v8 as a reference.